### DISCUSSION TITLE
timout during method call

### DISCUSSION THREAD

**Discussion by pfetrial** [FreeOpcUa/python-opcua#1238] [Q&A] [answered]
hello every body,
I wish a happy year!
Is there possible for a client to adjust the timout for a long method call? In my case, the call is a write a buffer into a file through opc ua . I have reduced the buffer size and transfer my file in little chunks . In this case no timeout is detected. The problem is when the client writes on the network ( a very slow one) , it is not the write operation server side.
Any other suggestions?
thx

**Comment by AndreasHeine** [comment, marked as answer]
sure! default is 4s

```python
client = Client(url, timeout)
```
https://github.com/FreeOpcUa/python-opcua/blob/0d595e596c9922699f0b36abb79583d5763eeb9b/opcua/client/client.py#L95

**Comment by pfetrial** [comment]
Oh great Thx

 

Von: Andreas Heine <notifications@github.com> 
Gesendet: Sonntag, 24. Januar 2021 09:14
An: FreeOpcUa/python-opcua <python-opcua@noreply.github.com>
Cc: pfetrial <patrick.frey37@sfr.fr>; Author <author@noreply.github.com>
Betreff: Re: [FreeOpcUa/python-opcua] timout during method call (#1238)

 

sure! default is 4s

client = Client(url, timeout)
